README.md

```diff
@@ -1,41 +1,49 @@
 [![Validate Primer](https://github.com/cooktheryan/gitops-primer/actions/workflows/validate-primer.yaml/badge.svg)](https://github.com/cooktheryan/gitops-primer/actions/workflows/validate-primer.yaml)
 
-# gitops-primer
-GitOps Primer is an operator can be deployed with a Kubernetes environment to export objects out of the cluster and store them within a Git repository.
+# snapshot-operator
+Snapshot Operator is an operator based on GitOps Primer (https://github.com/migtools/gitops-primer) that can be deployed 
```

Comment:
I don't have the full picture here but from what is happening in this PR I think the Snapshot Operator is not going to be based on the Gitops Primer but rather it is a fork of it, right? We're not going to pull from gitops-primer going forward or are we?

Reply:
Correct, I've changed the language here to make this more clear.